github-archive/launch.sh

```diff
@@ -12,13 +12,33 @@ if [[ -z "$2" ]] ; then
   exit 1
 fi
 
+function sed_replace() {
```

Comment:
you are right. however, it's getting complex if you want to replace a string with a variable that contains a string with "/" in it (like the aws secret access key) and you want it to edit in place. It is a lot easier when you create a temp file then.